### PR TITLE
Disallow Slack Invitations

### PR DESCRIPTION
**Pull request by MadMikeyB**
Temporarily disallow slack invitations to stop the influx of spambots.

### CODE REVIEW

**Comment by MadMikeyB** [comment]
@stevepop This follows discussion in the [#website](https://laraveluk.slack.com/archives/C4J630Y9Y/p1562166237014800) channel, and should just cut off all Slack Invitations at the source. Users are logged in case we want to manually invite them.

In future I suggest we implement a moderation queue for the invitations.

File: app/Helpers/Slack.php [hunk 1]
```diff
@@ -8,18 +8,14 @@
...
     public static function sendInvitation($email) {
-        //return \Unirest\Request::post(config('slack.invitation_url'), [], ['token' => config('slack.legacy_token'), 'email' => $email]);
-        $client = new \GuzzleHttp\Client();
-        $response =  $client->request('POST', config('slack.invitation_url'), ['query' => [
-            'token' => config('slack.legacy_token'),
-            'email' => $email
-        ]]);
-        $response = json_decode($response->getBody());
-        return $response;
+        Log::info($email . ' attempted to obtain an invitation to Slack');
```
Comment (Jamesking56) [inline app/Helpers/Slack.php]: IP address as well?

Comment (stevepop) [reply app/Helpers/Slack.php]: I think for now, Logging the email is sufficient as sometimes IP addresses might be VPN based.

Comment (MadMikeyB) [reply app/Helpers/Slack.php]: While there's nothing to stop us storing IP addresses, some do think it counts as PII, as discussed in Slack, but only when stored against another piece of PII.

Emails are more valuable to us than IP's at this point, so we just store those for now in the log. :)

**Pull request merged**